src/main/java/uk/gov/hmcts/ccd/domain/model/casedataaccesscontrol/RoleAssignmentAttributes.java

```diff
@@ -1,14 +1,19 @@
 package uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol;
 
+import java.util.Optional;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Builder
 @Data
+@NoArgsConstructor
+@AllArgsConstructor
 public class RoleAssignmentAttributes {
-    private String jurisdiction;
-    private String caseId;
-    private String region;
-    private String location;
-    private String contractType; // SALARIED, FEEPAY
+    private Optional<String> jurisdiction;
+    private Optional<String> caseId;
+    private Optional<String> region;
+    private Optional<String> location;
+    private Optional<String> contractType; // SALARIED, FEEPAY
```

Comment:
can you remove the comment please. It can go out of date if they start including new values so there's no point in adding it here. We can look at the endpoint API documentation if we want to know the values